cmd/spec/path.yaml

```diff
@@ -70,6 +73,9 @@ paths:
           application/json:
             schema:
               "$ref": "#/components/schemas/v1.PubkeyRequest"
+            examples:
+              example:
+                $ref: '#/components/examples/v1.PubkeyRequestExample'
         description: request body
         required: true
       responses:
```

Comment:
Once this PR is merged, we can keep adding the following example references and example values in the `pex` package.